apps/condo/domains/banking/schema/BankSyncTask.test.js

```diff
@@ -610,10 +610,6 @@ describe('BankSyncTask', () => {
             expect(obj.integrationContext.id).toEqual(bankAccount.integrationContext.id)
         }
 
-        beforeAll(async () => {
```

Comment:
Is it part of this PR?

Reply:
No, I just walked by and saw that an extra beforal was written. Decided to fix it right away